### DISCUSSION TITLE
Welcome to helix-gpt Discussions!

### DISCUSSION THREAD

**Discussion by leona** [leona/helix-gpt#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SilverCoder** [comment]
Hey there,

Just wanted to check in – is the project still active? It's been a while since there was any activity, and there are a few open pull requests hanging around.

Cheers!

**Reply by cameroncarlg** [reply]
Hey, also checking in on this. Maybe we can get some steam going around this project? Seems like a last step in getting people to switch from VSCode!
